vitrina/resources/migrations/0001_initial.py

```diff
@@ -0,0 +1,84 @@
+# Generated by Django 3.2.14 on 2022-09-14 06:48
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('vitrina_datasets', '0003_auto_20220905_0914'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='DistributionFormat',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created', models.DateTimeField(auto_now_add=True, null=True)),
+                ('modified', models.DateTimeField(auto_now=True, null=True)),
+                ('version', models.IntegerField()),
+                ('title', models.CharField(blank=True, max_length=255, null=True)),
+                ('deleted', models.BooleanField(blank=True, null=True)),
+                ('deleted_on', models.DateTimeField(blank=True, null=True)),
+            ],
+            options={
+                'db_table': 'distribution_format',
+                'managed': True,
```

Comment:
When runngin `manage.py migrate` I get following error: 

    psycopg2.errors.DuplicateTable: relation "distribution_format" already exists

Maybe

```suggestion
                'managed': False,
```

?

Same with other models.